fedimint-api/src/module/mod.rs

```diff
@@ -159,15 +160,41 @@ pub struct ApiEndpoint<M> {
 
 // <()> is used to avoid specify state.
 impl ApiEndpoint<()> {
-    pub fn from_typed<E: TypedApiEndpoint>() -> ApiEndpoint<E::State> {
+    pub fn from_typed<E: TypedApiEndpoint>() -> ApiEndpoint<E::State>
+    where
+        E::Param: Debug,
+        E::Response: Debug,
+    {
+        #[instrument(
+            target = "fedimint_server::request",
+            level = "trace",
+            skip_all,
+            fields(method = E::PATH),
+            ret,
+            err(Debug),
+        )]
+        async fn handle_request<'a, 'b, E>(
+            state: &'a E::State,
+            dbtx: fedimint_api::db::DatabaseTransaction<'b>,
+            param: E::Param,
+        ) -> Result<E::Response, ApiError>
+        where
+            E: TypedApiEndpoint,
+            E::Param: Debug,
+            E::Response: Debug,
+        {
+            tracing::trace!(target: "fedimint_server::request", ?param, "recieved request");
```

Comment:
This is too low, IMO.

Reply:
:shrug: I just specify `RUST_LOG="fedimint_server::request,...."` when I want to see the requests.

Reply:
@Maan2003 Can you PR a `docs/debugging.md` file with just this suggestion. We should try to start compiling some of these debugging tips and tricks.

Reply:
BTW. I also think `fedimint_server` here is not needed. We should just make it on of the categories. See #1469.